ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py

```diff
@@ -1497,3 +1497,30 @@ def test_tensorflow_abs(
         rtol=1e-02,
         x=x[0],
     )
+
+# sqrt
+@handle_frontend_test(
+    fn_tree="tensorflow.math.sqrt",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+    ),
+    test_with_out=st.just(False),
+)
+def test_tensorflow_sqrt(
+    *,
+    dtype_and_x,
+    on_device,
+    fn_tree,
+    frontend,
+    test_flags,
+):
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        rtol=1e-02,
+        x=x[0],
+    )
```

Comment:
ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py:1526:6: W292 no newline at end of file